DriverLoader.h

```diff
@@ -40,14 +40,28 @@
 namespace osvr {
 namespace vive {
     struct CouldNotLoadDriverModule : std::runtime_error {
+#if defined(OSVR_LINUX) || defined(OSVR_MACOSX)
+        CouldNotLoadDriverModule(const char *errString)
+            : std::runtime_error(
+                  "Could not load driver module." + 
+                   std::string(errString)) {}
```

Comment:
You should probably add a space between the hard-coded C-string and the appended error string.